client/src/components/CreateJob/createjob.js

```diff
@@ -65,6 +67,17 @@ class CreateJob extends Component {
           <Container fluid>
             <Row>
               <Col size="md-6">
+              <CreateJobModal
+            show={this.state.modalShow}
+            onHide={modalClose}
+            loadJobs={this.loadJobs}
+          />
```

Comment:
Looks like some indentations slight issues. linting/prettify would help with this. Not a big issue... But worth looking into at some point.